### ISSUE TITLE
[9.0][CI] project_timelog: request for unittests

### ISSUE THREAD

**Issue by GabbasovDinar** [itpp-labs/misc-addons#531]
The` 9.0` branch's `project_timelog` module should have valid python tests
`https://github.com/it-projects-llc/misc-addons/tree/9.0/project_timelog`

**Thorough specification of the task is the following:**

1. Go to "Project" in the menu and click "Tasks" in submenu
2. Click on "Task" in the kanban view
3. Go to "Timesheets" tab
4. Click the "Edit" button
5. Add a new item and specify the subtask name
6. Click the "Save" button
7. Run timer using button of play
8. Wait 5 secound
9. Stop timer using button of stop

**Additional information**

Instructions about odoo js tour and phantom js can be found here https://odoo-development.readthedocs.io/en/latest/qa/js/index.html

Python tests instruction is here https://odoo-development.readthedocs.io/en/latest/qa/python/index.html

**Comment by yelizariev** [comment]
Actually, there is a tour test, but it doesn't work for some reason.

**Comment by yelizariev** [comment]
Ok, it doesn't work, because it loads ``/``, while the tour is in backend

```
        self.phantom_js("/", "odoo.__DEBUG__.services['web.Tour'].run('project_timelog', 'test')", "odoo.__DEBUG__.services['web.Tour'].tours.project_timelog", login="admin")
```
@GabbasovDinar fix this please and activate the tour

**Comment by yelizariev** [comment]
fixed in #609
